### ISSUE TITLE
Improvement in speed of netTiers 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [netTiers/netTiers#389]
```
I added some features to improve the speed of the build process of our nettiers 
code.
Hopefully someone would implement this in the NetTiers.

here is the times before and after:

Initializing
Loading templates
Generation complete. Execution time: 00:01:55.5336082
You can view current documentation @ http://docs.netTiers.com
Generated solution can be found here: 
file:\\\C:\SourceControl\NeoMedNet\DAL\NeoMedNET.sln

After: 

Initializing
Loading templates
Generation complete. Execution time: 00:01:20.9736314
You can view current documentation @ http://docs.netTiers.com
Generated solution can be found here: 
file:\\\C:\SourceControl\NeoMedNet\DAL\NeoMedNET.sln

basically some of the changes are: 
1 thread to create the project \ solutions files
1 thread to create the tables files
1 thread to create the View files
```

Original issue reported on code.google.com by `TheBig...@gmail.com` on 16 Jun 2011 at 5:51
